modules/core/src/main/java/org/apache/ignite/internal/processors/metric/GridMetricManager.java

```diff
@@ -495,6 +508,21 @@ private long totalSysMemory() {
         }
     }
 
+    /**
+     * @return Bounds for the {@link #CACHE_OPERATIONS_BLOCKED_DURATION_HISTOGRAM} metric. 30 bounds with 1-second
+     * interval by default.
+     */
+    private long[] getDurationHistogramBounds() {
```

Comment:
Why this method?
Let's write
`long[] bounds = new long[] {
0, 1000, 2000, 3000,... 30000
}`

These bounds look too much for me, also.
Can we start with `500, 1000, 5000, 30000` intervals?
User will be able to configure this.